src/Microsoft.AspNet.Razor/Properties/AssemblyInfo.cs

```diff
@@ -5,5 +5,4 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("Microsoft.AspNet.Razor.Test")]
-[assembly: InternalsVisibleTo("Microsoft.AspNet.Razor.Runtime.Test")]
```

Comment:
With the `InternalsVisibleTo` `Microsoft.AspNet.Razor.Runtime` would never be able to use an `aspnet/Common` package that `Microsoft.AspNet.Razor` was using. 

Currently not an issue but definitely an unneeded restriction.